package.json

```diff
@@ -79,13 +80,7 @@
   },
   "xo": {
     "extends": [
-      "prettier"
```

Comment:
Simplified this config a bit, it's exactly the same: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration